frontend/hub/administration/repositories/RepositoryPage/RepositoryDetails.tsx

```diff
@@ -30,7 +30,7 @@ export function RepositoryDetails() {
     params.id ? pulpAPI`/distributions/ansible/ansible/?name=${params.id}` : ''
   );
   const { data: remoteData, error: remoteError } = useGet<Task>(
-    params.id ? pulpAPI`/remotes/ansible/collection/${parsePulpIDFromURL(repository.remote)}/` : ''
+    params.id ? pulpAPI`/remotes/ansible/collection${parsePulpIDFromURL(repository.remote)}/` : ''
```

Comment:
this will also create the wrong url, same issue as https://github.com/ansible/ansible-ui/pull/1552#discussion_r1472822781

Reply:
(this should not be `params.id ? ` but `repository.remote ? `...)